legacy/firmware/fsm_msg_common.h

```diff
@@ -491,7 +491,10 @@ void fsm_msgApplyFlags(const ApplyFlags *msg) {
 void fsm_msgRecoveryDevice(const RecoveryDevice *msg) {
   CHECK_PIN_UNCACHED
 
-  const bool dry_run = msg->has_dry_run ? msg->dry_run : false;
+  CHECK_PARAM(!msg->has_kind || msg->kind == 0 || msg->kind == 1,
+              _("UnlockRepeatedBackup not supported"))
+
+  const bool dry_run = msg->has_kind ? msg->kind == 1 /* RecoveryKind.DryRun */ : false;
```

Comment:
Use constants, which are generated by nanopb in legacy/firmware/protob/messages-bitcoin.pb.h.

Reply:
https://github.com/trezor/trezor-firmware/pull/3743/commits/65fef0e3af060fb95245121b8d6e50a194dad8cb#diff-c738534bc71383aea91a0922ce6a24165865374082a3d5775aa45b223fad62e8R497

Reply:
not done

Reply:
True :) https://github.com/trezor/trezor-firmware/pull/3743/commits/098497b989e63cd426ac770e3f16b70036960d1a#diff-c738534bc71383aea91a0922ce6a24165865374082a3d5775aa45b223fad62e8R494